packages/collections/list.pony

```diff
@@ -123,18 +123,26 @@ class List[A] is Seq[A]
   new create(len: USize = 0) =>
     """
     Do nothing, but be compatible with Seq.
+
+    Creates an empty list with 0 nodes.
+
+    let my_list = List[String].create()
```

Comment:
Standard style here would be to drop the create:
`let my_list = List[String]`
Perhaps there's another way to remind users about the create name?